<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="gauche_camera">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Gauche</string>
   </property>
  </widget>
  <widget class="QLabel" name="titre_vitesse">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>580</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Vitesse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hgauche">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>560</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Haut/Gauche</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="camera"/>
  </widget>
  <widget class="QPushButton" name="droite">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>610</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Doite</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connexion">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connexion</string>
   </property>
  </widget>
  <widget class="QSlider" name="slide_vitesse">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>620</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>120</number>
   </property>
   <property name="maximum">
    <number>240</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="bas_camera">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Bas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hdroite">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>560</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Haut/Droite</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Gauche">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>610</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Gauche</string>
   </property>
  </widget>
  <widget class="QLabel" name="titre_batterie">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Batterie</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="barre_batterie">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="droite_camera">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Droite</string>
   </property>
  </widget>
  <widget class="QPushButton" name="avancer">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Avancer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="haut_camera">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Haut</string>
   </property>
  </widget>
  <widget class="QLabel" name="IRDroit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IRDroit</string>
   </property>
  </widget>
  <widget class="QLabel" name="IRGauche">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>220</y>
     <width>49</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IRGauche</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reculer">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>610</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Reculer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bdroite">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>650</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bas/Droite</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bgauche">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>650</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bas/Gauche</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deconnexion">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>460</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Deconnexion</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>75</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLCDNumber" name="ir_AvD"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="ir_AvG"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLCDNumber" name="ir_ArD"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="ir_ArG"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>350</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="Odometrie">
      <property name="text">
       <string>Odo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="odo_gauche"/>
    </item>
    <item>
     <widget class="QLCDNumber" name="odo_droit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>510</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="detection_collisions">
      <property name="text">
       <string>Collisions</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="collision">
      <property name="text">
       <string>collision</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="screenshot">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>460</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Screenshot</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
